tests/unit/utils/test_ssdp.py

```diff
@@ -0,0 +1,531 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email:`Bo Maryniuk <bo@suse.de>`
+'''
+
+from __future__ import absolute_import, print_function, unicode_literals
+from tests.support.unit import TestCase, skipIf
+from tests.support.mock import (
+    NO_MOCK,
+    NO_MOCK_REASON,
+    MagicMock,
+    patch)
+
+from salt.utils import ssdp
+import datetime
+
+try:
+    import pytest
+except ImportError as err:
```

Comment:
This is a nitpick, but if we're not using `err`, we don't really need to save the `ImportError` instance. It can just be `except ImportError:`.